src/beliefpropagation/beliefpropagation_schedule.jl

```diff
@@ -0,0 +1,40 @@
+default_edge_sequence_alg() = "forest_cover"
+
+@traitfn default_bp_niters(g::NamedGraph::(!IsDirected)) = is_tree(g) ? 1 : nothing
+function default_bp_niters(g::AbstractGraph)
+  return default_bp_niters(undirected_graph(underlying_graph(g)))
+end
+
+function edge_sequence(g::NamedGraph; alg=default_edge_sequence_alg())
+  return edge_sequence(alg, g)
```

Comment:
```suggestion
  return edge_sequence(Algorithm(alg), g)
```
so that strings representing the algorithms can be passed as keyword arguments.